addons/pyroscope/readme.md

```diff
@@ -0,0 +1,195 @@
+# Pyroscope
+
+This addon is built based [Pyroscope](https://github.com/pyroscope-io/pyroscope), which consist of server and agent. It allows the user to collect, store, and query the profiling data in a CPU and disk efficient way.
+
+## install
+
+```shell
+vela addon enable pyroscope
+```
+
+## uninstall
+
+```shell
+vela addon disable pyroscope
+```
+
+## Note
+### Pyroscope for Golang applications
+
+- To start profiling a Go application, you need to include our go module in your app
+    ```shell
```

Comment:
What's the vela application using this trait look like? Can you put a demo in example? I truly suggest add a demo in example/ dir

Reply:
As described by https://github.com/kubevela/catalog/pull/361#issuecomment-1147019554